{"id": "5340", "title": "Bug 5340 - Cancelling add exception breakpoint has no effect", "description": "", "OB": "When attempting to cancel the process of adding an exception breakpoint, the progress dialog closes immediately. Despite the cancellation request, the exception list still appears, indicating that it ignored the cancellation. It is expected that the cancellation request is honored, and either the exception list should not appear, or the cancel button should be disabled. To achieve this, when calling ProgressMonitorDialog.run, pass false for the `cancellable` parameter.", "EB": "", "SR": ""}